### DISCUSSION TITLE
authentication log and fail2ban

### DISCUSSION THREAD

**Discussion by quamok** [TandoorRecipes/recipes#584] [Q&A]
Hello there, does anyone knows if there's an authentication log for failed login attempts somewhere that I could use fail2ban to ban IPs with too many failed logins?

**Comment by vabene1111** [comment]
There currently is no such feature but i am sure it would not be too hard to implement. Feel free to open an issue. Please also describe there what exactly it is that you need (just a table or also an API, etc.)
